lib/assets/javascripts/turbograft/turbolinks.coffee

```diff
@@ -106,33 +109,33 @@ class window.Turbolinks
 
     return
 
-  @loadPage: (url, xhr, partialReplace = false, onLoadFunction = (->), replaceContents = [], replaceAllExcept = []) ->
+  @loadPage: (url, xhr, options = {}) ->
```

Comment:
should url and xhr go into options hash here?
mainly because I noticed above `url` is passed in as `null`